CSSTooltipPseudo/explainer.md

```diff
@@ -0,0 +1,896 @@
+
+Unlike [`interesttarget`](
+https://open-ui.org/components/interest-invokers.explainer/#why-is-interesttarget-not-unlimited-like-title-is),
+the `title` attribute is allowed on any element, even if it isn't
+interactable. This can pose problems for accessiblity, because if the
+element isn't focusable, it means that the associated tooltip is not
+keyboard accessible.
+
+On top of this, no browser, outside of Microsoft Edge, exposes to
+built-in tooltip on focus, even if the associated element is focusable.
+This means that browser built-in tooltips are not accessible
+out-of-the-box, which is one of the goals of this proposal.
+
+### Proposed improvements
+
+To improve the current situation, the `::tootlip` element cannot be
+triggered or styled if the element associated with the tooltip is
+not focusable.
+
+As part of this effort, browser vendors should also work on triggering
+their built-in tooltips on focus when the element associated with the
+tooltip is focusable, and utilize the escape key to trigger tooltip
+dismissal.
+
+One consideration for improving this further is to force existing
+`title`-based tooltips to have a `tabindex` of `0` if it is not
+already focusable. However, this is not ideal, as it could lead
+a container to become focusable that was not before, leading to
+undesirable focus changes if a user incorrectly applies focus just
+outside of a target element within that container.
+
+An alternative for browser vendors to consider is including a user
+setting that forces a `tabindex` of `0` in such cases, allowing the
+user more control over the resulting behavior as they see fit.
+
+Creating a more accessible tooltip experience for users is one of
+the key goals of this proposal, so additional ideas from the
+community that should be considered to help in further improving
+the current situation is very much welcome.
+
+### Zoom
+
+Built-in tooltips today do not adhere to user preferred zoom levels,
+making them less accessible than the content they are associated with.
+
+Making these tooltips stylable through the new `::tooltip` pseudo
+element should allow for a more accessible experience in such
+scenarios.
+
+## Security Considerations
+
+As noted in the section on the [current landscape of built-in
+tootlips](#current-landscape-of-built-in-tootlips), browser built-in
+tooltips are currently allowed to escpape the bounds of the browser
+window.
+
+For tooltips styled via the `::tooltip` pseudo element, UAs **must** instead
+stay within the bounds of the web page. This would avoid [security concerns](
+https://textslashplain.com/2017/01/14/the-line-of-death/) when rendering
+outside the bounds of the browser window, which becomes exacerbated when
+styleability is involved, because this can more easily trick users into
+clicking on a spot on the screen that could lead to a security vulnerability.
+
+This behavior would be limiting for Iframes in particular but would be
```

Comment:
I was curious about the iframe behavior - with this proposal, tooltips would be limited to the content area of the iframe, right? Might be worth spelling out here.

Reply:
Added more details, thanks for the callout